tests/document_test.py

```diff
@@ -0,0 +1,26 @@
+import unittest
```

Comment:
As this is yamlidator test, this probably should be in `apluslms-yamlidator/tests`. I'm not sure, but filename could follow the module, e.g. `test_utils_collections.py`